packages-exp/auth-exp/src/core/persistence/persistence_user_manager.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Persistence, PersistedBlob } from '../persistence';
+import { User } from '../../model/user';
+import { ApiKey, AppName, Auth } from '../../model/auth';
+import { inMemoryPersistence } from './in_memory';
+import { UserImpl } from '../user/user_impl';
+
+export const AUTH_USER_KEY_NAME_ = 'authUser';
+export const PERSISTENCE_KEY_NAME_ = 'persistence';
+const PERSISTENCE_NAMESPACE_ = 'firebase';
+
+export function persistenceKeyName_(
```

Comment:
I don't think we have an official rule on prefix _ vs suffix _ for internal functions/constants, but I'd like to establish the convention of using prefix _. `@firebase/firestore` and `@firebase/app` are already using prefix _ today and the documentation tool we are going to use will [flag functions with `@internal` annotation without prefix _](https://api-extractor.com/pages/tsdoc/tag_internal/).  
WDYT?

Reply:
Sg, done